ethereum/contract_addresses.go

```diff
@@ -67,7 +67,7 @@ var ChainIDToContractAddresses = map[int]ContractAddresses{
 		ERC1155Proxy:        common.HexToAddress("0x7eefbd48fd63d441ec7435d024ec7c5131019add"),
 		Coordinator:         common.HexToAddress("0x38a795580d0f687e399913a00ddef6a17612c722"),
 		CoordinatorRegistry: common.HexToAddress("0x45797531b873fd5e519477a070a955764c1a5b07"),
-		DevUtils:            common.HexToAddress("0xccc2431a7335f21d9268ba62f0b32b0f2efc463f"),
+		DevUtils:            common.HexToAddress("0xd74ff1bfe071e4586c5ab969335a649e3bd9c518"),
```

Comment:
Ok, this explains your correction on my comment that David would need to update `@0x/contract-addresses`. He wouldn't because we have a Go version of this package.